index.js

```diff
@@ -54,6 +54,41 @@ function monkeypatch() {
     estraverse.VisitorKeys.TypeAlias = TypeAliasKeys;
     return results;
   };
+
+  // monkeypatch escope/referencer
+  var referencerLoc;
+  try {
+    var referencerLoc = Module._resolveFilename("./referencer", escopeMod);
+  } catch (err) {
+    throw new ReferenceError("couldn't resolve escope/referencer");
+  }
+  var referencer = require(referencerLoc);
+
+  // if there are decotators, then visit each
+  function visitDecorators(node) {
+    if (!node.decorators) {
+      return;
+    }
+    for (var i in node.decorators) {
```

Comment:
Haha I guess since we're actually monkeypatching it's a good assumption to make!